pkg/notifications/notifiers.go

```diff
@@ -51,25 +47,15 @@ func NewNotifier(notifierType string) (notifier Notifier, err error) {
 }
 
 // NewNotifiers returns a list of notifiers configured in the config file
-func NewNotifiers(configFile string) ([]Notifier, error) {
+func NewNotifiers() ([]Notifier, error) {
 
 	var notifiers []Notifier
 
-	if configFile == "" {
-		zap.S().Debug("no config file specified")
-		return notifiers, nil
-	}
-
-	configReader, err := config.NewTerrascanConfigReader(configFile)
-	if err != nil || configReader == nil {
-		return notifiers, err
-	}
-
 	// get config for 'notifications'
-	notifications := configReader.GetNotifications()
-	if notifications == nil {
-		zap.S().Debug("key '%s' not present in toml config", notificationsConfigKey)
-		return notifiers, ErrTomlKeyNotPresent
+	notifications := config.GetNotifications()
+	if len(notifications) == 0 {
+		zap.S().Debug("no notification detected from config")
```

Comment:
May be we can improve this message, something like `no notification specified in config`

Reply:
`no notifications specified in the config file`